simpleansi.go

```diff
@@ -18,15 +18,28 @@ func MoveCursor(row, col int) {
 
 const RESET = "\x1b[0m"
```

Comment:
I think RESET also should be made private.

Reply:
thank you; that is done.